x-pack/plugins/ml/public/application/components/anomalies_table/links_menu.tsx

```diff
@@ -452,7 +452,10 @@ export const LinksMenuUI = (props: LinksMenuProps) => {
           // Build query using categorization regex (if keyword type) or terms (if text type).
           // Check for terms or regex in case categoryId represents an anomaly from the absence of the
           // categorization field in documents (usually indicated by a categoryId of -1).
-          if (categorizationFieldType === ES_FIELD_TYPES.KEYWORD) {
+          if (
+            categorizationFieldType === ES_FIELD_TYPES.KEYWORD ||
+            categorizationFieldType === ES_FIELD_TYPES.VERSION
```

Comment:
Categorization is one place where we _shouldn't_ let people choose a `version` field.  Trying to categorize `7.17.3` and `8.2.1` will not work well.  All the values will end up with zero tokens going through to the inner categorizer as we ignore numbers.

Reply:
Removed in https://github.com/elastic/kibana/pull/132606/commits/63c9e1c025e7fdb391cbda61075318e44e8581f9